Jenkinsfile_nightly

```diff
@@ -2,7 +2,7 @@
 
 properties([
     // H allow predefined but random minute see https://en.wikipedia.org/wiki/Cron#Non-standard_characters
-    pipelineTriggers([cron('H 05 * * *')])
+    pipelineTriggers([cron('H 04 * * *')])
```

Comment:
May I change my advice to change this to 07, please?
This is to make data store after Def Store, and keep 2 hours difference against random factor in minutes.

Reply:
Hour changed to 07 as suggested.